draft-scenarios/coops/close_to_enemies.tex

```diff
@@ -0,0 +1,222 @@
+\end{itemize}
+
+\textbf{Town Buildings:} \svgunit{bronze} Dwelling.
+
+\textbf{Map Tile Pool:} Take $P$ × random Far (II–-III) Map Tiles to form a shared pool, where $P$ equals the number of players. None of these Tiles may contain a Settlement.
+
+\subsection*{\MakeUppercase{Map Setup}}
+
+Take the following Map Tiles and arrange them as shown in the Scenario map layout.
+Place Far (II--III) Map Tiles with Settlements that match your Faction's Towns. Mark these with a colored Cube (see example).
+
+\begin{itemize}
+  \item $P$ × Starting (I) Map Tile.
+  \item $P$ × Far (II--III) Map Tile with Settlement of same Faction as players Towns.
+  \item 2$P$ × Far (II--III) Map Tile - must not contain Tiles with Settlements.
+\end{itemize}
+
+\textit{Optional: You can distribute random pieces and covered pieces on the map so that they match your Faction's basic terrain.}
+
+\subsection*{\MakeUppercase{AI Hero Setup}}
+
+Prepare an AI Deck for Settlement defenders.
+
+\textbf{Settlement defenders Deck:} 1 × Attack Card, 1 × Defense Card, 1 × Spell Card
+
+\textbf{Settlement defenders Spell Deck:} 1 × Magic Arrow
+
+\subsection*{\MakeUppercase{Victory Conditions}}
+Conquer all Settlements.
+
+\subsection*{\MakeUppercase{Defeat Conditions}}
+
+\begin{itemize}
+  \item Any player loses their Town.
+  \item At least one Settlement remains unclaimed by the end of the \nth{7} Round. Time has run out!
+\end{itemize}
+
+\subsection*{\MakeUppercase{Timed Events}}
+    \textbf{\nth{1} Round:}
+    \begin{itemize}
+      \item At start of the Round, each player show up the \textit{nemesis map Tiles} (Far (II--III) Map Tile with Settlement of same Faction as players Towns). This Tile you can reveal freely.
+    \end{itemize}
+    \textbf{\nth{2} Round:}
+    \begin{itemize}
+      \item  First wave of enemy clan armies attacks the towns. (see below)
+      \item  Each player who defended his own town, during this Round can hire secondary Hero for 5 \svg{gold}, but you still have to use the population token.
+    \end{itemize}
+    \textbf{\nth{3} Round:}
+    \begin{itemize}
+      \item At start of the Round, each player shows up one Tile (Far (II--III) Map Tile. This Tile you can reveal freely. (Not matter if Tile is from pool or map template.)
+    \end{itemize}
+    \textbf{\nth{4} Round:}
+    \begin{itemize}
+      \item Second wave of enemy clan armies attacks the towns. (see below)
+      \item Each player who defended his own town, search (3) the \textit{Artifact Deck}.
+     \end{itemize}
+    \textbf{\nth{6} Round:}
+    \begin{itemize}
+      \item Third wave of enemy clan armies attacks the towns. (see below)
+      \item Each player who defended his own town, search (3) the \textit{Artifact Deck}.
+    \end{itemize}
+    \textbf{\nth{7} Round:}
+    \begin{itemize}
+      \item During this Round, Players can discard and draw Cards at the start of their turn twice rather than once.
+    \end{itemize}
```

Comment:
```suggestion
\end{itemize}
\textbf{\nth{7} Round:}
\begin{itemize}
  \item During this Round, each player may perform the discard and draw action twice at the start of their turn, instead of the usual once.
\end{itemize}
```